Fix rest provider example for kc >= 23

cdevienne opened the pull request:
Starting with KC23, a `@Path` is required for an exposed method to work properly

Fixes #26088

ahus1 commented:
Thank you for the pull request. According to our contributing guidelines, your commits need to be signed off. 

See `CONTRIBUTING.md` in this repository for details, or https://www.keycloak.org/2023/10/dco.html for additional details.

cdevienne committed:
Fix rest provider example for kc >= 23

Starting with KC23, a @Path is required for an exposed method to work properly

Fixes #26088

Signed-off-by: Christophe de Vienne <christophe.devienne@orus.io>

cdevienne commented:
Sorry I missed the part about signing off. I forced-pushed a new changeset.
